Libraries/test/EventsTests.Shared/EventTests.cs

```diff
@@ -1933,6 +1933,87 @@ private void Handle(KafkaEvent kafkaEvent)
             }
         }
 
+        [Theory]
```

Comment:
Is this change integration tested?

Reply:
Could you please elaborate? I didn't setup ActiveMQ or RabbitMQ cluster and configured it with Lambda. I simply used guidance from sample Lambda events at https://docs.aws.amazon.com/lambda/latest/dg/with-mq.html.

Reply:
Tested end-to-end using instructions at https://awscloudfeed.com/whats-new/compute/using-amazon-mq-as-an-event-source-for-aws-lambda (reused the same instructions for Active MQ and Rabbit MQ).

**NOTE:** For RabbitMQ, there was intermittent error where it was unable to deserialize priority. The error went away after creating a test Node.js Lambda function to examine the event JSON (it reported `partition` as integer value). After some testing, the error in .NET Lambda went away.

Reply:
I don't even see `partition` in the event models. has it been missed?

Reply:
Sorry, it's `priority`. I was working on multiple items in parallel.